package.json

```diff
@@ -5,6 +5,7 @@
   "workspaces": [
     "apps/*",
     "packages/*",
+    "packages/embeds/*",
```

Comment:
@zomars doesn't packages/* cover this? I see you did the same with app-store so I'm probably misunderstanding workspaces.

Reply:
Are there going to be multiple embeds as different packages? If not, then is is unnecessary. Also unless there are multiple embeds and share a lot of common code I would rather have different packages: `packages/embed-1`, `packages/embed-2`, etc.